ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/IntegrationTestBase.java

```diff
@@ -335,4 +343,34 @@ private static File createFile(final String s) throws IOException {
     file.deleteOnExit();
     return file;
   }
+
+  private static int httpJsonRpcPort() {
```

Comment:
nit: I can't help but feel that the flow of the calling function should be:
awaitPortsFile(fileName);
int port = extractPortFromFile(fileName);

Rather than having the "Await" hidden down here... (super minor)

Reply:
done